### DISCUSSION TITLE
Filters on side of table list

### DISCUSSION THREAD

**Discussion by helloreece** [filamentphp/filament#10595] [Ideas]
The filters on table components are already extremely powerful. I'd love to be able to give them the space they deserve. See a mock up here.

I'm not technically gifted enough to create this, but putting it here in case others are looking for this feature also.

![Example](https://github.com/filamentphp/filament/assets/129170707/77687458-cbcf-4ccd-a33f-5db762b6f393)

**Comment by edeoliv** [comment]
Wow man this is amazing I love this mock up

**Comment by danharrin** [comment]
https://filamentphp.com/docs/4.x/tables/filters/layout#displaying-filters-to-the-left-or-right-of-the-table-content
